frontend/src/redux/exams/reducers.js

```diff
@@ -21,13 +24,20 @@ export default combineReducers({
         return action.exams;
       case ACTIONS.SET_GET_EXAM_SUCCESS:
         return action.exam;
-      case ACTIONS.SET_POST_EXAM_SUCCESS:
-        return [
+      case ACTIONS.SET_PATCH_EXAMS_SUCCESS:
+        return action.exams;
+        case ACTIONS.SET_POST_EXAM_SUCCESS:
+        return state.exams.length ? {
+            ...state,
+            exam: action.exam,
+            exams: [
```

Comment:
Can you clarify why we need both `exam` and `exams` property on this state object in the singular post_exam_success?

Reply:
Because otherwise I need to check which is my new exam in the exams array. Here I know that "exam" is the new one and I can send it's params in the route !